src/index.js

```diff
@@ -24,50 +24,79 @@ const editTodo = (i, descr) => {
   window.location.reload();
 };
 
+const clearCompleted = () => {
+  const itemsToDelete = [];
+
+  todo.getAll().forEach((item) => {
+    if (item.completed) {
+      itemsToDelete.push(item);
+    }
+  });
+
+  itemsToDelete.forEach((item) => {
+    todo.delete(item.index);
+  });
+
+  localStorage.setItem('mydata', JSON.stringify(todo.getAll()));
+  window.location.reload();
+};
+
```

Comment:
- [ ] You have done a good job here :muscle: :clap: but the `clear all completed` is slightly malfunctioning by not deleting exactly the number of tasks completed. Kindly check the [this video](https://www.awesomescreenshot.com/video/15307630?key=d9eede031a2fcf95f26d06b08c3b8c12) to have a grasp of this behavior.
- Using `Array.filter()` might make the sorting of completed tasks easier, kindly see if it suits you well :pray: 
- [ ] [OPTIONAL] It might help modularize your code more so that it can be clean. Consider creating a file for each `CRUD` functions in the module directory, and then import then into your `index.js` file. This will really make your code more maintainable, suitable for testing, and allows others to understand your code even better :thought_balloon: